### ISSUE TITLE
Unable to add devices anymore (collection topic)

### ISSUE THREAD

**Issue by TedTolboom** [TedTolboom/com.xiaomi-mi-zigbee#222]
Please share in this issue, if you experience issues with not being able to add Zigbee devices anymore.

Please note: the Xiaomi / Aqara app does not control or influence the inclusion of a Zigbee device. 
It only provides the inclusion instructions and after a match has been found, it will take over the device with it's corresponding driver.

Please share the following information:
1. Homey version currently installed
2. Screenshot of the Zigbee devices shown at https://developer.athom.com/tools/zigbee
3. Homey diagnostics log (Settings > General > Create Diagnostics Report
4. Brief description of the issue; specifically the amount of devices you tried to add since the last reboot

**Comment by robiebab** [comment]
I have problems adding the humidity sensor. When i try to include the sensor(press long until is see 3 times blue light), nothing happens(i also don’t see the device in my zigbee network). 

Homey version: 5.0.0-rc.34

Report id:  5c0f13b9-0bb0-4058-ba91-3ead972c2df5

I have 3 zigbee devices (xioami contact sensor) that already been there

**Comment by TedTolboom** [comment]
@robiebab thanks for reporting, please the Homey diagnostics report as well; so Athom can follow-up

**Comment by robiebab** [comment]
Thanks, Ted! Homey diagnostic report id: 8254678EE1

**Comment by nissenhj** [comment]
1. Homey version currently installed
 5.0.-rc.34
2. 
![zigbee](https://user-images.githubusercontent.com/45319576/94890483-a0333400-047f-11eb-92ca-1303e11f91f5.jpg)
3.  Homey diagnostics log
2CDAD49BD3
4a. Aqara app diagnostics log
98d6f573-51d6-4012-b138-18c7fb92c2a2
4b. hue app diagnostics log
3ad6301b-4512-4a83-b251-c626fe53fb73
5. Brief description of the issue; specifically the amount of devices you tried to add since the last reboot
Unable to install Zigbee devices since upgrading to version 5.
Tried to install 3 units. 1. hue motion sensor 2. aqara motion sensor 3. aqara vibration sensor.
Nothing happens when trying to add them.

**Comment by TedTolboom** [comment]
@nissenhj can you try to unplug and replug after 2 minutes the routine devices (Osram plugs and Hue bulbs) at the order of distance from Homey (closest first)? See if this rebuilds the mesh of routine devices.

Once this is done, please restart Homey and see if you can add devices again.

**Comment by nissenhj** [comment]
Yesterday evening I try so with the closest device, the hue bulb. Nothing happened

**Comment by jerroenbakker** [comment]
1. Homey 5
2. ![image](https://user-images.githubusercontent.com/31915919/94990688-55e4ac80-057e-11eb-9d70-1d18b09562d4.png)
3. 288BABBA9C
4. 12bdb7fa-2f95-4b82-9fce-34dd864a8a30
5. Tried to install aqara sensors (door, body). Nothing happens when trying to add them.

**Comment by dirkg173** [comment]
1. v5.0.0-rc.34
2. 
![1D8B9CFD-1148-4755-AF1E-A40C075129E1](https://user-images.githubusercontent.com/40319629/94991860-45383480-0586-11eb-9ce6-75aee7a4632b.jpeg)

3. Homey Diagnostic Report: CBC7B555D6
4. App Diagnostic Report: ffbb594b-3d59-4588-8c44-aa5234ec9cae
5. See #223 and [Link](https://community.athom.com/t/aqara-xiaomi-smart-home-zigbee-app-v0-8-0-test-v1-0-1/156/2150?u=fantross)

**Comment by Anton6401** [comment]
1. Homey V5.00 RC34
2.
![screenshot-developer athom com-2020 10 04-10_08_48](https://user-images.githubusercontent.com/72335296/95010550-8aae3d80-062a-11eb-8826-509763c2c622.png)

3. Homey diagnostic report. C057BA1F4F
4. App diagnostic report, 7dce29ba-5f5b-45b6-b2d4-5ffac5786d64

5.No Zigbee devices can be added, tried a new Aqara temperature sensor, a removed Aqara temperature sensor and tried a removed INNR ligtbulb RT248.
There are also 2 Aqara temperature sensors and 1 Aqara motion sensor not responsive.

**Comment by Anton6401** [comment]
Just found out that the Aqara door & window sensors can be added.
So not all Aqara devices have this problem.

**Comment by AR-JEN** [comment]
1. 5.0.0-rc.34

2. Screeenshot:
![image](https://user-images.githubusercontent.com/44006275/95132138-90596f80-075f-11eb-80b8-a16628252cac.png)

3. Homey diagnostics log: 
99DA078262

4. App diagnostic report
88e0409d-9bae-466c-af34-c4d1c481201f

5. One Aquara Motion sensor worked after the App update, a second one didn't work. I removed the second one, but after that I am not able to add the sensor to the Zigbee network again. Same for one Double remote wall switch (WXKG02LM)

**Comment by dirkg173** [comment]
> 1. v5.0.0-rc.34
> 
> ![1D8B9CFD-1148-4755-AF1E-A40C075129E1](https://user-images.githubusercontent.com/40319629/94991860-45383480-0586-11eb-9ce6-75aee7a4632b.jpeg)
> 
> 1. Homey Diagnostic Report: CBC7B555D6
> 2. App Diagnostic Report: ffbb594b-3d59-4588-8c44-aa5234ec9cae
> 3. See #223 and [Link](https://community.athom.com/t/aqara-xiaomi-smart-home-zigbee-app-v0-8-0-test-v1-0-1/156/2150?u=fantross)

**Update:** Hey @TedTolboom, today I tried again to add the Aqara Wireless Mini Switch (2018). This time directly at the place of use (about 3 meters away from Homey). On the second try it worked. But I had to take care that I release the button immediately after the third flash. On the first try I was too slow and the LED flashed 4 times, so it didn't work.
The Zigbee mesh looks good and the switch is connected over another device (lamp).

**Comment by itshows** [comment]
1: 5..0.0-rc.34
2:  
![afbeelding](https://user-images.githubusercontent.com/46216331/95656412-94dea900-0b0e-11eb-9842-079d9cbebc9f.png)
3: BD7C776065
4: 95be563a-d2c2-4177-a97c-a7506f20319f
5: when i try to add a aqara door/window sensor, nothing happens. Install screen in app does not change whatever i try. It is a door/window sensor where ypu have to put in a pin to reset the device. All my existing aqara zigbee devices work perfectly, including some door/window sensors.

**Comment by jaccostraaijer** [comment]
Homey version currently installed: 5.0.0-rc.34

Screenshot of the Zigbee devices shown at https://developer.athom.com/tools/zigbee
![image](https://user-images.githubusercontent.com/32959702/95755822-27b14c00-0ca5-11eb-8a71-f3f9485d554f.png)

Homey diagnostics log (Settings > General > Create Diagnostics Report)
9A75A6C5AF

Aqara app diagnostics log (Settings > Apps > Aqara app > Settings > Create Diagnostics Report)
a77790ec-5cbf-41ab-91e1-11f79c08475e

Brief description of the issue; specifically the amount of devices you tried to add since the last reboot
I restored a backup. After the backup 2 humidity sensors were not working anymore. One is in the screenshot with a red square around the route (second node in the route is a ?. I deleted the other and tried to re-add without success.

**Comment by TedTolboom** [comment]
Thanks for the feedback all!

Athom, inclusion of devices is handled in Homey's core and not the Aqara app, is currently looking into the issue and working to resolve it.

I'll keep you posted if I get an update.

**Comment by seblang** [comment]
Homey version 500rc34
#
Homey report
B74D8CF3BD

Aqara diag
38666537-3f41-4a99-bffa-ba9cc9f861e9

Issue :
Insertion of light sensor impossible[](url

**Comment by nissenhj** [comment]
Anyone made it work after the update to version 5.0.0-rc.38?  
No difference at all to me!

**Comment by AR-JEN** [comment]
> Anyone made it work after the update to version 5.0.0-rc.38?
> No difference at all to me!

Also here no difference after RC.38

**Comment by StefanO-Kjell** [comment]
1. 5.0.0-rc.38
2.
![image](https://user-images.githubusercontent.com/24807945/96724807-86b34700-13b0-11eb-9885-f684f6a82edc.png)
 
3. Homey diagonstic: E2E961E57C
4. Aqara diagnostic: 26aa09fc-3ad6-4847-b83a-bd2b861520e7
5. Installed version 5.0.0-rc.38 and removed 1 of 3 of my former installed Aqara Motion Sensors. I then included it again without no problem (see screenshot). So I continued to do this with the rest of the sensors. But on the last one (3 of 3) I could not get Homey to include it.

**Comment by gdekeijzer** [comment]
Still having problems add motion sensor.
Still needing to supply info?

1) Homey 5.0 rc38
2) 
![image](https://user-images.githubusercontent.com/3164475/96725540-3f798600-13b1-11eb-96a1-3863b736aa86.png)
3) Diiagnostic report: E22678EF1E
4) Aqara app: 6ef5406e-21a3-4656-8656-cc1498f0ee78
5) No devices added since last reboot and after upgrade to 5.0

**Comment by gdekeijzer** [comment]
> Edit 3: Got a response from Athom:
> 
> "Thank you for reporting this issue, and our apologies for your inconvenience!
> 
> This issue is already reported by other users and Development is working hard to fix this issue.
> 
> We hope to resolve this problem with the next update of the Homey firmware.
> Best Regards,
> 
> Boaz
> 
> The Athom support team."

Yeah, got that reply too. When asking when this was going to be released they said "in the next few weeks".
Ok, i'll hold...

**Comment by vova275** [comment]
1. 5.0.0-rc.38
2. 
3. E0451D6DC3
4. 68e54eff-715c-400b-b9bf-7f017a4fc305
5. none of my three temperature sensors are added.

**Comment by vova275** [comment]
![20201025_135158](https://user-images.githubusercontent.com/40322484/97106286-4feb7280-16c9-11eb-9029-7e10482c27d6.jpg)

**Comment by TedTolboom** [comment]
In addition to Athom support (Boaz') feedback, also a brief update by Athom Development:
> **Robin Bolscher 13:36**
> We are aware of an issue introduced with Homey v5.0.0 which can prevent Zigbee devices from joining the network in certain situations. We know why this happens in some situations and are currently working on a fix for this.

As always, no timeline mentioned by Athom

**Comment by shaarkys** [comment]
I was able to add additional several Aqara devices facing this issue - but always PTP was required and pairing in close distance (FYI)

**Comment by Sjalabert** [comment]
@TedTolboom 
5.0.0-rc.38

Screenshot:
![image](https://user-images.githubusercontent.com/36713703/98341867-d2e8c300-200f-11eb-818e-5a8b827b5acf.png)

Homey diagnostics log:
3D9598A01C

App diagnostic report
f51b-4049-8ceb-f4cdcd859f65

Not possible to add the 'Aqara opple' two and six buttons. (also not innr-SP120, but is antoher app)

**Comment by TheArcturian** [comment]
I just want to add that it is impossible to add the Aquara Cube. It does get detected when trying to add it, but after that it eventually times out (even though I press the button every 2-3 seconds as instructed).

**Comment by nissenhj** [comment]
> I just want to add that it is impossible to add the Aquara Cube. It does get detected when trying to add it, but after that it eventually times out (even though I press the button every 2-3 seconds as instructed).

Have you updated to RC43?

**Comment by TheArcturian** [comment]
No. I'm running the latest official version. Homey is already too unstable in my setup so going for a release candidate is not an option. I'd rather wait.

**Comment by nissenhj** [comment]
> No. I'm running the latest official version. Homey is already too unstable in my setup so going for a release candidate is not an option. I'd rather wait.

This topic is for 5.0.0-rc.(38)

**Comment by nissenhj** [comment]
> No. I'm running the latest official version. Homey is already too unstable in my setup so going for a release candidate is not an option. I'd rather wait.

Try RC43 it´s run great!

**Comment by TheArcturian** [comment]
Hmm, is there any known issues in RC43 that might break something? Like an app or whatever.

**Comment by shaarkys** [comment]
@TheArcturian please note that your question is OT. If you are not familiar with Experimental versions, don't update yet (https://community.athom.com/t/homey-v5-0-0-erfahrungen-und-kompatible-apps/32560/284). If you are already on RC38 and consider upgrade to RC43 - I don't see any reason why not to update.

**Comment by TheArcturian** [comment]
I cant see where this is off topic. Teds first post does not refer to RC-versions anywhere or in the title.

**Comment by MrR0bert** [comment]
I just received my Xiaomi Light Sensor but Homey can't detect the app that is needed for it to add this device.

The requested information:

1. Homey version currently installed
4.2.0
3. Screenshot of the Zigbee devices shown at https://developer.athom.com/tools/zigbee
![image](https://user-images.githubusercontent.com/20636874/101169749-b2774d00-363d-11eb-8b24-ea02972f550f.png)
4. Homey diagnostics log (Settings > General > Create Diagnostics Report)
F11B348408
5. Aqara app diagnostics log (Settings > Apps > Aqara app > Settings > Create Diagnostics Report)
9c400bd4-fa68-409a-8ee9-9affae9ed025
6. Brief description of the issue; specifically the amount of devices you tried to add since the last reboot
I just received this sensor and after adding it to Homey, I receive the message that no supporting app could be found and that has been added as an "uknown zigbee device".

Also, I took the liberty of browsing the code and did find a driver for this sensor...so I have no idea why it isn't working.

![image](https://user-images.githubusercontent.com/20636874/101170080-36c9d000-363e-11eb-98cd-9338d4d0ebb9.png)

**Comment by MrR0bert** [comment]
Some additional information, when I run the app with the cloned GitHub code my sensor does work.

**Comment by shaarkys** [comment]
I think it work just because of APP was newly started.

**Comment by MrR0bert** [comment]
@shaarkys thanks but I doubt that's the case. Since I also tried to restart Homey and reinstalled the app from the store. Both would, in my opinion, result in a restart/clean start of the app. But only the cloned code recognises the sensor.

**Comment by technik242** [comment]
Hello and me too,
i cant add zigbee devices (hue bulb and gledopto)
Diagnostic Report 8C93D7A8AF,
Hue Zigbee report b49ff517-123f-477c-9005-7283a6e3006b
gledopto report a5d68287-8f5b-4fe1-8a4c-80809c567b9d

The devices are found and then get the app a timeout.


Greetings from Germany.

**Comment by Patrick1610** [comment]
Hi, I have the problem that when adding a Light Sensor, the app says 'No compatible Homey app has been found for this device. It has been added as a Basic Zigbee device.' The basic zigbee device is present in Homey after that, but doesn't do anything. It is just a button, no tags or values present. 

Is my problem related to this one, or should I make another issue for that error?

**Comment by jseverinson** [comment]
1. Currently running version 5.0.0

2. 
<img width="1783" alt="zigbee" src="https://user-images.githubusercontent.com/4082348/107820510-3acf4800-6d7b-11eb-9fbd-585b9b9a80b0.png">

3. Report ID: 8FFCE1

4. 807a8c75-50ad-463c-aa5a-df3c03af0a4b

5. I’ve had two Aqara Wireless Remote Switch (Double Rocker) installed. A couple of weeks ago one stopped working, tried removing it but couldn’t re-pair it. Tried switching battery, still no dice. When pairing the wizard shows instructions and then starts pairing, but nothing happens after and eventually it will say ”Adding a device took too long, no device was found”. Figured something was off with this switch (the other one was still working) so I brought out a third one I had yet to unpack. However it’s the same with this device, the pairing starts but can’t complete so something seems to be off that is not a fault with the devices.

**Comment by KoenHak** [comment]
Homey version currently installed
5.0.1

<img width="1543" alt="image" src="https://user-images.githubusercontent.com/26246832/111024447-1d73b500-83df-11eb-9834-31a6e343c4cd.png">

Homey diagnostics log (Settings > General > Create Diagnostics Report)
2D389F

Aqara app diagnostics log (Settings > Apps > Aqara app > Settings > Create Diagnostics Report)
b7eb7e7c-fd79-42fa-adcd-a5d3d554a926

Brief description of the issue; specifically the amount of devices you tried to add since the last reboot
I tried to add several devices, like the Aqara D1 Single, Aqara D1 Double Switch, Innr GU10 229. I can't add them. The install process is started, but finished with a time out after a few minutes ("Het toevoegen duurde te lang, geen apparaat gevonden.")

**Comment by epjjansen** [comment]
You only have end devices added.
Remove some of them. Then add routers like bulbs or plugs. Next add the end devices again

**Comment by KoenHak** [comment]
Thanks. I was under the impression that I could add max 15 devices. But I removed 3 Aqara Temp sensors and now I could add the D1 double. Not the single, neither the Innr GU10, but at least this is a good starting point. I hoped I could add the GU10, to have a router added to the Zigbee network, but not successfull yet. 

Thanks for your advise! 👍🏻

**Comment by Albinosaurus** [comment]
Homey version currently installed 7.1.0
Screenshot of the Zigbee devices shown at https://developer.athom.com/tools/zigbee
Homey diagnostics log (Settings > General > Create Diagnostics Report): 128A2D
Aqara app diagnostics log (Settings > Apps > Aqara app > Settings > Create Diagnostics Report) ff241db0-cf7b-4df7-ba56-e8504aa4dcc5
Brief description of the issue; specifically the amount of devices you tried to add since the last reboot. Trying to add a Wireless Switch WXKG11LM. Nothing happens while in pairing mode. Tried two of them and all 3 options in the app (Wireless Switch, Wireless Switch 2018, Wireless Mini Switch). 

![afbeelding](https://user-images.githubusercontent.com/87931092/126895290-c514defd-463f-424c-8fcb-d529eebdc736.png)

**Comment by Albinosaurus** [comment]
Burrp

**Comment by TedTolboom** [comment]
Now you honestly think I am motivated to respond to your issue / question?

**Comment by Albinosaurus** [comment]
Well I guess not.
